src/docs/background/10-custom-pricing/index.md

```diff
@@ -0,0 +1,263 @@
+whereas a day-based action respectively calculates the number of days. A
+unit-based action just takes a _quantity_ parameter that it uses to
+construct the line item that holds the information about the number of
+booked units.
+
+Every line item has a unit price and one of the following attributes:
+_quantity_ and _percentage_. The quantity attribute can be used to
+denote the number of booked units, like number of booked nights. The
+percentage param is used when modeling commissions for example. Based on
+these attributes a line total is calculated for each line item. Line
+totals then define the total payin and payout sums of the transaction.
+
+When using a night, day and unit based pricing actions the Flex API
+constructs the line items based on parameters passed in a request. The
+marketplace website or app does not need to concern how the line items
+are created but it can rely on the information they contain when
+presenting the transaction information in the UI.
+
+## More complex pricing schemes with custom pricing
+
+In addition to a pricing scheme that is based on booking length or
+number of booked units, pricing can be extended or completely
+re-designed by using _custom pricing_. Custom pricing works so that it
+uses a specific custom pricing action called `set-line-items-and-total`
+in the transaction process. As the name suggests, this action allows you
+to define any number of line items you need. So the client application,
+that is your marketplace website or application, passes the booking
+related line items to the API when initiating a transaction. When
+passing line items to the API, a unit price and a quantity or a
+percentage need to be defined for each line item. Then based on these,
+the API calculates a total for each line item and the overall total sums
+for the transaction.
+
+When using the custom pricing action, it is to be noted that it can not
+be combined with the night, day or unit-based pricing actions that rely
+on the API to construct the line items based on booking length or a
+number of units. So if the intention is to design the pricing so that
+the booking length and a number of add ons affect the pricing, also the
+line item that describes the actual booking needs to be passed to the
+API by the client. Commissions and price negotiations can be combined
+with custom pricing but the price of the actual booking needs to consist
+of the line items provided by the client application.
+
+To see how custom pricing works in practice let's look at an example of
+initiating a transaction using the
+[Flex JavaScript SDK](https://github.com/sharetribe/flex-sdk-js). In
+this example we'll create a booking with the following description:
+
+- a booking for a hotel room for three nights for \$80 per night
+- a baby crib as an additional service for \$5 per night
+- a discount of 15%
+
+In addition the marketplace process defines a customer commission of 10%
+that is calculated in the API.
```

Comment:
The code example has provider commission so that's what this should probably say too.